Tests/PlotTests/HTMLTests.swift

```diff
@@ -529,6 +529,30 @@ final class HTMLTests: XCTestCase {
         let html = HTML(.comment("Hello"), .body(.comment("World")))
         assertEqualHTMLContent(html, "<!--Hello--><body><!--World--></body>")
     }
+
+    func testPicture() {
+        let html = HTML(.body(
+            .picture(
+                .source(
+                    .src("/img"),
+                    .srcset("/img 2x"),
```

Comment:
Shouldn't this be a valid URL, instead of a string containing a space?

Reply:
Good comment. I actually need to change the type of this to a string. What is actually expected is a common separated list of image/pixel density pairs[1].
```
<img
 srcset="
  /wp-content/uploads/flamingo4x.jpg 4x,
  /wp-content/uploads/flamingo3x.jpg 3x,
  /wp-content/uploads/flamingo2x.jpg 2x,
  /wp-content/uploads/flamingo1x.jpg 1x
 "
 src="/wp-content/uploads/flamingo-fallback.jpg"
>
```
[1] - https://html.com/attributes/img-srcset/

Reply:
Cool, you could probably model that using an array of strings, right?